onnxruntime/python/tools/symbolic_shape_infer.py

```diff
@@ -1151,6 +1153,28 @@ def _infer_aten_pool2d(self, node):
             elem_type = onnx.TensorProto.INT64 if i == 1 else self.known_vi_[node.input[0]].type.tensor_type.elem_type
             vi.CopyFrom(helper.make_tensor_value_info(o, elem_type, get_shape_from_sympy_shape(sympy_shape)))
 
+    def _infer_aten_minmax(self, node):
+        dim_or_y = self._try_get_value(node, 1)
+        keepdim = self._try_get_value(node, 2)
```

Comment:
None doesn't mean input[1] and input[2] don't exist, it can be some non-constant inputs. I think use len(node.input) is enough to switch the cases, if 1 is max, if 3, then max.dim. If len(node.input) is 3 and dim or keepdims is None, it means unkown axis, it's still max.dim, there are some code in _infer_ReduceSum you can borrow to handle such case.